src/main/java/com/comcast/drivethru/api/HTTPRequestManager.java

```diff
@@ -418,6 +441,17 @@ public Builder data(byte[] data)
             mData = data;
             return this;
         }
+        
+        /**
+         * Sets request's multipart data.
+         * @param multipart Multipart data to send the server
+         * @return {@link Builder} object
+         */
+        public Builder multipart(HttpEntity multipart)
```

Comment:
Please add a `@see` annotation that will point to `MultipartEntityBuilder` api to help users create this multi-part entity